pydm/widgets/spinbox.py

```diff
@@ -21,6 +21,7 @@ def __init__(self, parent=None, init_channel=None):
         self.setEnabled(False)
         self._alarm_sensitive_border = False
         self._show_step_exponent = True
+        self._limits_from_pv = True
```

Comment:
We are trying to not use the name `PV` anymore... `Channel` is the proper name for PyDM. Do you mind to change: `_limits_from_pv` -> `_limits_from_channel`